_posts/2017-03-05-artsy-technology-stack-2017.md

```diff
@@ -0,0 +1,100 @@
+---
+layout: post
+title: Artsy's Technology Stack, 2017
+date: 2017-03-05
+categories: [Technology, eigen, force, gravity]
+author: orta
+series: Artsy Tech Stack
+---
+
+Artsy has now grown past 160 team members and our Engineering organization is now 30? strong. [WIP] For a brief overview of what the company has accomplished in the last few years, check out our [2013](http://2013.artsy.net) and [2014](http://2014.artsy.net) reviews.
+
+I'd like to comprehensively cover what, and how we make the technical side of Artsy work.
+
+<!-- more -->
+
+# Organizational Structures
+
+In 2016, we made considerable changes to [the Engineering organizational stack](/blog/2016/03/28/artsy-engineering-organization-stack/). This is still an accurate representation of our team structure.
```

Comment:
I would add a summary, _We continue to have a Product Engineering organization that focuses on delivering value to customers with the help of self-sufficient and vertical product teams, and a Platform Engineering organization that enables horizontal efficiency, common infrastructure and practices._